index.js

```diff
@@ -2195,6 +2196,36 @@
         return filter(is(type), Just(x));
       });
 
+  //. ### Numeric
+
+  //# add :: FiniteNumber -> FiniteNumber -> FiniteNumber
+  //.
+  //. Adds two finite numbers together.
```

Comment:
For the descriptions, I like:

  - _Returns the sum of two (finite) numbers._
  - _Returns the difference between two (finite) numbers._